app/src/App.js

```diff
@@ -62,7 +63,7 @@ const App = () => {
   }
 
   return (
-    <Main>
+    <Main theme={appearance} assetsUrl="./aragon-ui">
```

Comment:
As i mentioned in another PR i don't think the `assetsUrl` is needed since `./aragon-ui` it's already the default